STUDIO/org.openl.rules.webstudio/webapp/css/common.css

```diff
@@ -1716,3 +1716,20 @@ option[value=""]:disabled + option[value=""]:disabled {
 .center-align {
     text-align: center;
 }
+
+.autocomplete-items {
+    position: absolute;
+    border: 1px solid #d4d4d4;
+    border-bottom: none;
+    border-top: none;
+    z-index: 99;
+}
+.autocomplete-items div {
+    padding: 5px;
+    cursor: pointer;
+    background-color: #fff;
+    border-bottom: 1px solid #d4d4d4;
+}
+.autocomplete-active {
+    background-color: #FAE5C1 !important;

```

Comment:
What is the reason for using !important here?
Can it be done with higher specifity in style declaration instead?

Reply:
It's required and it doesn't work without this tag.